lib/jobs/message_latency_report.ex

```diff
@@ -30,14 +30,15 @@ defmodule Jobs.MessageLatencyReport do
       ) do
     Enum.each(0..(days_to_analyze - 1), fn diff ->
       date = start_date |> Date.add(-diff) |> Date.to_string()
-      Logger.info("Generating message latency report for #{date}...")
+      Logger.info("message_latency_report: Generating message latency report for #{date}...")
```

Comment:
If we want this prefix everywhere, perhaps we should create a helper function that applies it. Also there's one `Logger.info` in `format_csv_data` that didn't get the prefix.